libs/libssh/patches/001-compile.patch

```diff
@@ -1,35 +1,39 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 5e603c2..28a44f1 100644
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -35,10 +35,6 @@ include(DefineInstallationPaths)
- include(DefineOptions.cmake)
- include(CPackConfig.cmake)
+@@ -35,10 +35,6 @@ include(GNUInstallDirs)
+ 
+ include(CompilerChecks.cmake)
  
 -# disallow in-source build
 -include(MacroEnsureOutOfSourceBuild)
 -macro_ensure_out_of_source_build("${PROJECT_NAME} requires an out of source build. Please create a separate build directory and run 'cmake /path/to/${PROJECT_NAME} [options]' there.")
```

Comment:
I would remove this section of the patch and use

CMAKE_BINARY_SUBDIR:=build in the Makefile, as is done elsewhere.

Reply:
Thanks. Done.